pkg/pillar/hypervisor/xen.go

```diff
@@ -517,8 +519,21 @@ func (ctx xenContext) Delete(domainName string) (result error) {
 	return nil
 }
 
+// getTaskStateFilePath returns file where xen-start puts state of domain
+// must be aligned with xen-start logic
+func (ctx xenContext) getTaskStateFilePath(domainName string) string {
+	return filepath.Join("/run/tasks", domainName)
+}
+
 // Cleanup removes containerd-shim
```

Comment:
This comment isn't correct with your changes.

Reply:
Correct, fixed, thank you